modules/node_modules/@ncigdc/components/ProjectsCharts.js

```diff
@@ -305,8 +302,14 @@ const ProjectsChartsComponent = compose(
         </div>
         {!genesIsFetching ? [
           <div style={{ alignSelf: 'center', color: theme.greyScale7, fontSize: '1.2rem' }} key='bar-subtitle'>
+            <ExploreLink
+              query={{
+                searchTableTab: 'cases',
+                filters: { op: 'and', content: filters },
```

Comment:
filters should either be `null` or the full filter object so you should be able to just do `{ filters }`.
shouldn't this also use `massagedFilters`?